examples/manual-example.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: "kubeflow.org/v1alpha1"
+kind: StudyJob
+metadata:
+  namespace: kubeflow
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: manual-example
+spec:
+  studyName: manual-example
+  owner: crd
+  optimizationtype: maximize
+  objectivevaluename: Validation-accuracy
+  optimizationgoal: 0.99
+  requestcount: 10
```

Comment:
return 10 same trial?
maybe we should only consider ignore requestcount for manual suggestion?

Reply:
Yes, the requestcount will be ignored for the manual suggestion.
I will delete it from example and add doc.

Reply:
rewrite both of requestNumber and requestCount to 1 when suggestion algo is manual.
https://github.com/kubeflow/katib/pull/352/commits/608f3970994cb7b6ac376f39a6d22f6e12986e25#diff-7a381fdaeae1295724b74c0a04f7644cR38